src/main/java/teammates/ui/webapi/action/ActionFactory.java

```diff
@@ -79,6 +79,7 @@
         map(ResourceURIs.LOCAL_DATE_TIME, GET, GetLocalDateTimeInfoAction.class);
         map(ResourceURIs.JOIN, GET, GetCourseJoinStatusAction.class);
         map(ResourceURIs.JOIN, PUT, JoinCourseAction.class);
+        map(ResourceURIs.JOIN_REMIND, POST, SendReminderEmailAction.class);
```

Comment:
`SendReminderEmailAction` -> `SendJoinReminderEmailAction`.

Let's make it clear as reminder email can be multiple things in our application.